src/managers/conda/condaUtils.ts

```diff
@@ -267,6 +267,14 @@ function nativeToPythonEnv(
         const shellDeactivation: Map<string, PythonCommandRunConfiguration[]> = new Map();
         shellActivation.set('gitbash', [{ executable: pathForGitBash(conda), args: ['activate', 'base'] }]);
         shellDeactivation.set('gitbash', [{ executable: pathForGitBash(conda), args: ['deactivate'] }]);
+        const condaRoot = path.dirname(path.dirname(conda));
+        shellActivation.set('zsh', [
+            { 
+                executable: 'source', 
+                args: [`"${path.join(condaRoot, 'etc', 'profile.d', 'conda.sh')}" && conda activate base && clear`] 
+            }
+        ]);
```

Comment:
You should split the `&&` out. This will help us construct it correctly in various scenarios
something like this:
```
[{ 
    executable: 'source', 
    args: [path.join(condaRoot, 'etc', 'profile.d', 'conda.sh')]
}, {
    executable: 'conda',
    args: ["activate", "base"]
}, {
    executable: 'clear',
    args: []
}]
```